usaspending_api/api_contracts/contracts/v2/bulk_download/awards.md

```diff
@@ -90,10 +90,7 @@ This route sends a request to the backend to begin generating a zipfile of award
 ## Filter Objects
 
 ### Filters (object)
-+ `agencies` (optional, array[Agency], fixed-type)
-All requests must either include the `agencies` or `agency` parameter.
-+ `agency` (optional, string)
```

Comment:
Should also be able to remove `+ sub_agency (optional, string)`

Reply:
@BrianZito is it valid to remove this filter from the API contract? If not a brief response would be appreciated

Reply:
Good catch